### ISSUE TITLE
A little point about the comment in model.py

### ISSUE THREAD

**Issue by xzjcs1992** [multimodallearning/pytorch-mask-rcnn#29]
In model.py line 884
The comment is
`# Reshape to [batch, 2, anchors]`
I think it might be [batch, anchors, 2]
That's why I have been confused about Softmax Layer with dim = 2

Anyway, the code is pretty good.

**Comment by eng100200** [comment]
@xzjcs1992  could you tell how this working and where can i get the nms_kernel.cu.o?

cd nms/src/cuda/
 nvcc -c -o nms_kernel.cu.o nms_kernel.cu -x cu -Xcompiler -fPIC -arch=[arch]
 cd ../../
 python build.py
 cd ../
